client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -44,13 +53,23 @@ const toggleSidebarShortcut = {
 
 registerBlockType( name, settings );
 
-const onboardingBlock = createBlock( name, {} );
-
 export function Gutenboard() {
 	const [ isEditorSidebarOpened, updateIsEditorSidebarOpened ] = useState( false );
-
 	const toggleGeneralSidebar = () => updateIsEditorSidebarOpened( isOpen => ! isOpen );
 
+	const onboardingState = useSelect( select => select( STORE_KEY ).getState() );
+
+	// FIXME: Quick'n'dirty step state, replace with router
+	const [ currentStep, setStep ] = useState( Steps.IntentGathering );
+	const goToNextStep = stepCompleted[ currentStep ]( onboardingState )
+		? () => setStep( step => step + 1 )
+		: undefined;
+	const goToPrevStep = currentStep > 0 ? () => setStep( step => step - 1 ) : undefined;
```

Comment:
Maybe its just nitpicking but I see the back button is always enabled in design. Not a blocker though.
```suggestion
	const goToPrevStep = currentStep > 0 ? () => setStep( step => step - 1 ) : () => window.history.back();
```

Reply:
I'd rather not go back to `window.history.back()`; I think it's reasonable to expect the user to only use the in-app 'Back' button to navigate within the flow (and not to use it to navigate back to wherever they were coming from). We might change this once we're more tightly integrated into Calypso, but I think it's fine to disable for now :grimacing: